mdx-models/src/main/java/com/mx/path/model/mdx/model/account/AccountAddress.java

```diff
@@ -0,0 +1,128 @@
+package com.mx.path.model.mdx.model.account;
```

Comment:
I don't see a reason to define AccountAddress. Isn't this just the same as Address? If we just use the same Address and avoid duplicating that code, could we get the same result? This would allow you to simplify Resources.java as well.

Reply:
I thought about it, both are very similar but there was a field apply_to_all that is only available for the Address model but not for the new account address model, which is why I went with a new model.